tools/nodejs/test/close_hang.test.ts

```diff
@@ -0,0 +1,39 @@
+import 'chai-as-promised';
+import {exec as _exec} from "child_process";
+import fs from "fs-extra";
+import path from "path";
+import {Database, OPEN_READWRITE} from "..";
+import {promisify} from "util";
+import {expect} from "chai";
+
+const exec = promisify(_exec);
+
+it("close hang", async function main() {
+  if (process.platform == 'win32') this.skip();
```

Comment:
Am I correct, that in windows it is still not resolved?

Reply:
It's just this test that can't run on windows

Reply:
Yep, I see that. I'm just wondering about the reasoning behind that. As this issue is reproducing on my Windows for version 0.9.2, so I was researching if it is about me to do smth wrong. Or the issue is not fixed for Windows.

Reply:
The issue is fixed for windows. This test uses lsof, which is is not available on windows